test/test_main.py

```diff
@@ -5,64 +5,45 @@
 import semver
 from datetime import datetime
 
-this_test_path = os.path.dirname(os.path.realpath(__file__))
-# Calculate absolute path of the CLI
-cli_path = os.path.join(this_test_path, '..', 'arduino-cli')
 
-# Useful reference: 
-# http://docs.pyinvoke.org/en/1.2/api/runners.html#invoke.runners.Result
-
-
-def cli_line(*args):
-    # Accept a list of arguments cli_line('lib list --format json')
-    # Return a full command line string e.g. 'arduino-cli help --format json'
-    cli_full_line = ' '.join([cli_path, ' '.join(str(arg) for arg in args)])
-    return cli_full_line
-
-
-def run_command(*args):
-    result = run(cli_line(*args), echo=False, hide=True, warn=True)
-    return result
-
-
-def test_command_help():
+def test_command_help(run_command):
     result = run_command('help')
     assert result.ok
     assert result.stderr == ''
     assert 'Usage' in result.stdout
 
 
-def test_command_lib_list():
+def test_command_lib_list(run_command):
     """
     When ouput is empty, nothing is printed out, no matter the output format
     """
     result = run_command('lib list')
     assert result.ok
     assert '' == result.stderr
-    result = run_command('lib list', '--format json')
```

Comment:
this was the only function calling `run_command` with more than one arg, I've merged the two into a single list to match the rest of the code